### ISSUE TITLE
Chat timestamp and name overlap at 125% scale.

### ISSUE THREAD

**Issue by metalgearsloth** [Regalis11/Barotrauma#9828]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Chat names overlap with the timestamp at > 100% text scales, see repro steps for what I did.

Pic:
![image](https://user-images.githubusercontent.com/31366439/188298776-ff5f30a4-b6e9-4e46-9a79-0a19cb203692.png)

### Reproduction steps

1. Set window resolution to 3840 x 2160
2. Set text scale to 125%
3. Chat names now overlap with the timestamp.

### Bug prevalence

Happens every time I play

### Version

0.18.15.1 / 0.18.15.2 on macOS

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

This seems to only affect messages already present on the when changing the text scale "on the fly", but everything appears to work correctly when e.g. setting the scale in the main menu before starting the game. I'd consider this a low priority issue.
